pkg/common/configs/configvalidator.go

```diff
@@ -79,6 +79,9 @@ func checkResource(res map[string]string) (int64, error) {
 		if err != nil {
 			return 0, fmt.Errorf("resource parsing failed: %v", err)
 		}
+		if rescount < 0 {
+			return 0, fmt.Errorf("not valid negative resource found")
```

Comment:
Can we add the negative value to the message for easier tracing: 
`"resource value invalid, cannot be negative: %d", rescount`